src/game/game.ts

```diff
@@ -83,23 +90,55 @@ export function finishTurn(state: GameState): GameState {
     3: 78° - 86°, 94° - 102°
     2: 70° - 78°, 102° - 110°
   */
-export function getTurnScore(state: GameState) : number {
 
-  const absDifference = Math.abs(state.turn.target - state.turn.guess)
+export function getGuessScore(state: GameState): number {
+  let absDifference = 0;
 
-  if(absDifference <= 8) {
+  if (
+    Math.sign(state.turn.target) === -1 &&
+    Math.sign(state.turn.guess) === 1
+  ) {
+    absDifference = Math.abs(state.turn.guess - state.turn.target);
+    console.log("this 2");
+  } else {
+    absDifference = Math.abs(state.turn.target - state.turn.guess);
+    console.log("this 3");
+  }
+
+  if (absDifference < 5) {
     return 4;
-  } else if(absDifference <= 24) {
+  } else if (absDifference < 13) {
     return 3;
-  } else if(absDifference <= 40) {
+  } else if (absDifference <= 25) {
     return 2;
   } else {
+    return 0;
+  }
+}
+
+/**
+ *
+ * @param state
+ * @returns
+ */
+export function getRebuttalScore(state: GameState): number {
+  let correctRebuttal = "";
+
+  if (state.turn.guess > state.turn.target) {
+    correctRebuttal = "right";
+  } else if (state.turn.guess < state.turn.target) {
+    correctRebuttal = "left";
+  }
```

Comment:
lol I misunderstood. And it was doing it so that the other team had to guess if the psychic's team guess was to the left or right of the target, not the target being to the left or right of the guess. Just updated!